agent/src/cli/jsonrpc.ts

```
@@ -166,11 +166,10 @@ class CodyPersister extends FSPersister {
                 // in in pull requests.
                 try {
                     const text = JSON.parse(responseContent.text)[0]
-                    console.log({ text })
                     const decodedBase64 = decodeCompressedBase64(text)
                     ;(responseContent as any).textDecoded = decodedBase64
-                } catch (error) {
-                    console.error('base64 decode error', error)
+                } catch {
+                    // console.error('base64 decode error', error)
```

Is that commented out on purpose?

---

Yes. Added a comment explaining why we can ignore this error.